tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_settings_RTL_POC.py

```diff
@@ -0,0 +1,173 @@
+        settings.return_to_main_menu()
+
+        settings.open_message_settings()
+        self.take_screenshot('message_settings')
+        settings.return_to_main_menu()
+
+        settings.open_cell_and_data_settings()
+        self.take_screenshot('cell_and_data')
+        settings.return_to_main_menu()
+
+        settings.open_bluetooth_settings()
+        self.take_screenshot('bluetooth')
+        settings.return_to_main_menu()
+
+        settings.open_internet_sharing_settings()
+        self.take_screenshot('internet_sharing')
+        settings.return_to_main_menu()
+
+        settings.open_sound_settings()
+        self.take_screenshot('sound')
+        settings.return_to_main_menu()
+
+        settings.open_display_settings()
+        self.take_screenshot('display')
+        settings.return_to_main_menu()
+
+        settings.open_homescreen_settings()
+        self.take_screenshot('homescreen')
+        settings.return_to_main_menu()
+
+        settings.open_search_settings()
+        self.take_screenshot('search')
+        settings.return_to_main_menu()
+
+        settings.open_navigation_settings()
+        self.take_screenshot('navigation')
+        settings.return_to_main_menu()
+
+        settings.open_notification_settings()
+        self.take_screenshot('notification')
+        settings.return_to_main_menu()
+
+        settings.open_date_and_time_settings()
+        self.take_screenshot('date_time')
+        settings.return_to_main_menu()
+
+        settings.open_language_settings()
+        self.take_screenshot('language')
+        settings.return_to_main_menu()
+
+        settings.open_keyboard_settings()
+        self.take_screenshot('keyboard')
+        settings.return_to_main_menu()
+
+        settings.open_theme_settings()
+        self.take_screenshot('themes')
+        settings.return_to_main_menu(gaia_header=False)
+
+        settings.open_addon_settings()
+        self.take_screenshot('addons')
+        settings.return_to_main_menu()
+
+        settings.open_achievements_settings()
+        self.take_screenshot('achievements')
+        settings.return_to_main_menu()
+
+        settings.open_firefox_accounts_settings()
+        self.take_screenshot('firefox_accounts')
+        settings.return_to_main_menu()
+
+        settings.open_findmydevice()
+        self.take_screenshot('findmydevice')
+        settings.return_to_main_menu()
+
+        settings.open_screen_lock_settings()
+        self.take_screenshot('screenlock')
+        settings.return_to_main_menu()
+
+        settings.open_app_permission_settings()
+        self.take_screenshot('app_permissions')
+        settings.return_to_main_menu()
+
+        settings.open_do_not_track_settings()
+        self.take_screenshot('donottrack')
+        settings.return_to_main_menu()
+
+        settings.open_browsing_privacy_settings()
+        self.take_screenshot('browsing_privacy')
+        settings.return_to_main_menu()
+
+        # This is a link to separate app
```

Comment:
In this case, I don't think we have to cover it in the Settings tests. Let's just put a comment saying why Privacy Control is not here